.github/workflows/build-multiarch-image-experimental.yml

```diff
@@ -32,5 +32,5 @@ jobs:
         with:
           context: .
           push: true
-          tags: ehrbase/ehrbase:yugabyte-experimental
+          tags: ehrbase/ehrbase:yugabyte-experimental2
```

Comment:
Why are we changing the tag of the image?

Reply:
because its not backwards compatible and we said we do that then.